### DISCUSSION TITLE
Is there any reason why most UI labels are not properly capitalized? (Most start with a lowercase letter)

### DISCUSSION THREAD

**Discussion by younesaassila** [benawad/dogehouse#621] [General]
I'm wondering why most UI labels/buttons start with a lowercase letter instead of an uppercase letter.

**Comment by BronzW** [comment]
More of a meme format

**Reply by BenNeighbour** [reply]
Yeah I guess so

**Reply by ghost** [reply]
or maybe It's something common but according to research people are into doing that and there's no specific reason for that :D

**Comment by DGKSK8LIFE** [comment]
if they are exported, they should be capital cased, if they are not, they should be named via lowerCamelCase @younesaassila

**Reply by younesaassila** [reply]
Sorry I'm not very familiar with specific programming terminology, what do you mean by "if they are exported"? Shouldn't they always be properly capital cased?

**Reply by DGKSK8LIFE** [reply]
Meaning if you `export {whatever}` or `export default whatever` in javascript, you name whatever you're exporting like so: `ExampleName`, but if they are locally scoped, or not exported, you are supposed to use this casing: `exampleLowerCamel`.

**Reply by younesaassila** [reply]
Oh sorry if I wasn't clear in my original message. I was referring to localized strings, not button variable names.

Like for example, why are some labels "voice settings" and not "Voice Settings"
